Gopkg.toml

```diff
@@ -145,7 +145,10 @@ required = [
   name = "k8s.io/client-go"
   version = "kubernetes-1.10.4"
 
-
+[[prune.project]]
+  name = "github.com/giantswarm/k8scloudconfig"
+  unused-packages = false
```

Comment:
I wonder if we can drop this line, i.e. `unused-packages = false`.

Reply:
Nope, I tried that. The only possible way to bypass that - add go file into each folder and import packages somewhere.

Reply:
That makes sense. It should be better when we have release-operator in place anyway.